frontend/src/components/pages/topics/TopicConfiguration.tsx

```diff
@@ -133,15 +133,18 @@ const ConfigurationEditor: FC<ConfigurationEditorProps> = observer((props) => {
                                             {$state.modalValueType === 'custom' && <>
                                                 <Text fontWeight="bold">Set at topic configuration</Text>
                                                 <Box maxWidth={300}>
-                                                    <ConfigEntryEditor className={'configEntryEditor ' + (isCustom ? '' : 'disabled')} entry={configEntry}/>
+                                                    ....{configEntry.frontendFormat}
+                                                    <ConfigEntryEditor
+                                                        className={'configEntryEditor ' + (isCustom ? '' : 'disabled')}
```

Comment:
Can/should we use `clsx` or `classnames` in these kinds of situations? @rikimaru0345 this question might be better answered by you, it's just a style guide we want to have for conditional classnames. Ideally we don't use classnames at all, so maybe we can just disregard this for now and fix later.

Reply:
I'm all for using `clsx` or `classNames`, I just didn't migrate it here.